playbooks/roles/deploy-osh/defaults/main.yml

```diff
@@ -79,7 +79,7 @@ suse_osh_deploy_always_redeploy: True
 # included in all the places).
 # For a more surgical per component approach, use service overrides, like
 # suse_osh_deploy_cinder_yaml_overrides for cinder for example.
-suse_osh_image_version: "ocata"
+suse_osh_image_version: "rocky"
```

Comment:
I guess we should create multiple variables, as it will be different if it's memcached or an openstack component.

Maybe:

```
suse_openstack_image_version: "rocky-opensuse_15"
suse_nonopenstack_image_version: "latest-opensuse_15"
```

?

Of we need to adapt all the references were this is used...

Shouldn't this be done organically when doing the overrides in the helm charts instead, though?